.docker/ci-testing/Dockerfile

```diff
@@ -2,7 +2,11 @@
 # CI image using the ROS testing repository
 
 ARG ROS_DISTRO=rolling
+<<<<<<< HEAD
 FROM osrf/ros2:testing
+=======
+FROM moveit/moveit2:${ROS_DISTRO}-ci
+>>>>>>> 39d371453 (Temporarily switch to dockerhub for CI jobs (#1908))
 LABEL maintainer Robert Haschke rhaschke@techfak.uni-bielefeld.de
 
 ENV TERM xterm
```

Comment:
Should this also incorporate the changes from #2035?

```suggestion
FROM osrf/ros2:testing
LABEL maintainer Robert Haschke rhaschke@techfak.uni-bielefeld.de

ENV TERM xterm

# Overwrite the ROS_DISTRO set in osrf/ros2:testing to the distro tied to this Dockerfile (OUR_ROS_DISTRO).
# In case ROS_DISTRO is now different from what was set in osrf/ros2:testing, run `rosdep update` again
# to get any missing dependencies.
# https://docs.docker.com/engine/reference/builder/#using-arg-variables explains why ARG and ENV can't have
# the same name (ROS_DISTRO is an ENV in the osrf/ros2:testing image).
ARG OUR_ROS_DISTRO=rolling
ENV ROS_DISTRO=${OUR_ROS_DISTRO}
RUN rosdep update --rosdistro $ROS_DISTRO
```

Reply:
If the changes from #2035 are made here as well, the corresponding github workflow build arg should be renamed to `OUR_ROS_DISTRO`: https://github.com/ros-planning/moveit2/blob/mergify/bp/humble/pr-1908/.github/workflows/docker.yaml#L134